test/input/yaml/CustomInput.yaml

```diff
@@ -100,4 +100,7 @@ parity_parameters:
     parity_check_file_index_col: feature_id
     parity_check_file_value_col: streamflow
     parity_check_compare_node: 4186169
+data_assimilation_parameters:
+    wrf_hydro_channel_ID_routelink_file: "../../test/input/geo/routelink/routeLink_subset.nc"
+
```

Comment:
For now, let's omit this reference, which will break in any test environment not on our workstation. We will need to use a separate PR to introduce a repository-hosted test case for the usgs direct replacement procedure.